docs/current_docs/manuals/developer/functions.mdx

```diff
@@ -256,37 +378,3 @@ Hello, world!
 /usr/local/bin #
 ```
```

Comment:
@vikram-dagger, the prompts in this block don't look good on web. The `shell` language is more for a shell script (file with code). That's why GitHub also has `console` for interacting with a terminal. That's why using `shell` here makes the `#` prompt appear like a line comment which hurts readability.

A normal _pre_ without a language marker would perheps be better, but maybe try https://prismjs.com/plugins/command-line/ (from https://prismjs.com/#plugins)?

Reply:
I looked into this a bit but it looks like we may be blocked since we use prism-react-render and it does not support laoding plugins: https://github.com/FormidableLabs/prism-react-renderer/issues/2